tools/update-lib-index.js

```diff
@@ -13,7 +13,6 @@ const fs = require('fs')
 const path = require('path')
 const { FlatESLint } = require('eslint/use-at-your-own-risk')
 const rules = require('./lib/rules')
-const configs = require('./lib/configs')
```

Comment:
Is that file still needed with your changes? Then please remove it.

Reply:
addressed in b88edc82238703965e11633c5b58e268e16bd238.